nixos/modules/services/games/openarena.nix

```diff
@@ -0,0 +1,61 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  name = "OpenArena";
```

Comment:
Why this abstraction?

Reply:
Over-eager variable extraction? I've inlined this variable now.